config/application.yml.example

```diff
@@ -1,3 +1,44 @@
+# These settings are common across environments, but may be overridden
+# in any particular environment
+aamva_verification_request_timeout: '50'
+aamva_verification_url: https://verificationservices-cert.aamva.org:18449/dldv/2.1/online
+account_reset_wait_period_days: '0'
+acuant_timeout: '55'
+acuant_simulator: 'true'
+acuant_sdk_document_capture_enabled: 'true'
+allow_doc_auth_test_credentials: 'true'
+attribute_encryption_key_queue: '["old-key-one", "old-key-two"]'
+aws_region: us-west-2
+cac_proofing_enabled: 'true'
```

Comment:
ditto .... this is an IDP feature flag, I think we should remove it here

Reply:
Yeah, I think there are quite a few IdP feature flags in here 🤔

Reply:
Ugh.. I think I mixed up the idp application.yml!! 🤦